qa_forum/views.py

```diff
@@ -0,0 +1,36 @@
+import json
+from django.core import serializers
+from django.http import HttpResponse
+from django.shortcuts import render, redirect
+from django.views.decorators.csrf import csrf_exempt
+from qa_forum.forms import QuestionForm
+from qa_forum.models import Question, Answer
+
+
+def index(request):
```

Comment:
@chuckcwh You should probably have a different route for the POST request instead of doing an if/else statement. This is highly undesirable.